### PR TITLE
[ricos-types]

### PR DESCRIPTION
**Pull request by oded-soffrin**
WIP

### CODE REVIEW

File: packages/ricos-common/web/src/biCallbacks/mediaUploadBI.ts [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import type { MediaUploadError } from 'wix-rich-content-common';
+import type { MediaUploadError } from 'ricos-content';
```
Comment (danarish4) [inline packages/ricos-common/web/src/biCallbacks/mediaUploadBI.ts:1]: Is there a reason why this type doesn't sit in `ricos-types` package?

File: packages/ricos-common/web/src/index.ts [hunk 2]
```diff
@@ -5,4 +5,4 @@ export { RicosEngine } from './RicosEngine';
 export { default as themeStrategy } from './themeStrategy/themeStrategy';
 export { default as localeStrategy } from './localeStrategy/localeStrategy';
 export { DRAFT_EDITOR_PROPS } from './consts';
-export type { DraftContent, RicosContent } from 'wix-rich-content-common';
+export type { DraftContent, RicosContent } from 'ricos-content';
```
Comment (danarish4) [inline packages/ricos-common/web/src/index.ts:8]: Why not ricos-type?

**Review by danarish4 — CHANGES_REQUESTED** [review]
MOSHLAM :)
Except for the fact I didn't understand why `DraftContent, RicosContent, MediaUploadError` are not inside ricos-types package

**Comment by oded-soffrin** [comment]
@danarish4 they were already in `ricos-content` so they shouldn't be an issue for shahar
I bet there are tons more types we can move, this is just to unblock

**Review by danarish4 — APPROVED** [review]

**Comment by shahardavid** [comment]
This PR os for removing `wix-rich-content-common` from `ricos-common`?

**Pull request merged**